src/main/scala/com/lightbend/lagom/docs/DocumentationGenerator.scala

```diff
@@ -29,16 +29,16 @@ object DocumentationGenerator extends App {
 
   // This impacts what gets displayed on the main documentation index.
   val stableVersions = Seq(
-    VersionSummary("1.2.x", s"Lagom $currentLagomVersion (current stable release) for Scala 2.11"),
-    VersionSummary("1.1.x", s"Lagom 1.1.0 (previous stable release) for Scala 2.11")
+    VersionSummary("1.2.x", s"Lagom $currentLagomVersion (current stable release)"),
+    VersionSummary("1.1.x", s"Lagom 1.1.0 (previous stable release)")
   )
 
   val previewVersions = Seq(
     VersionSummary("1.3.x", s"Lagom 1.3.0-M1 for Scala 2.11")
```

Comment:
Hmmm, that won't be trivial. Those sub-headings are built dynamically from the docs on lagom/lagom repo. We should go to the main repo, (for each branch) create a new file with the string we want to feed into the subheading and then read it from `DocumentationGenerator`. Finally, update `documentationVersionIndex.scala.html` to display that. Doable, but I don't think it's worth the effort until we start supporting Scala 2.12 or Java 9.